docs/api/covidcast-signals/quidel.md

```diff
@@ -54,7 +65,7 @@ meaningful levels starting May 26, 2020.
 | `covid_ag_smoothed_pct_positive_age_65plus` | Percentage of antigen tests that were positive for COVID-19 (ages 65+), smoothed by pooling together the last 7 days of tests. <br/> **Earliest date available:** 2020-05-26 |
 | `covid_ag_smoothed_pct_positive_age_0_17` | Percentage of antigen tests that were positive for COVID-19 (ages 0-17), smoothed by pooling together the last 7 days of tests. <br/> **Earliest date available:** 2020-05-26 |
 
-### Estimation
+## Estimation
```

Comment:
Quidel headings were all +1 from what the other indicator pages had, so I pushed them back 1.